### ISSUE TITLE
Level 9 Tables

### ISSUE THREAD

**Issue by Jacquelyn-Rudis** [YMCACOLUMBUS/activescience-issues#49]
In hypothesis section
Table should have arrow indicating which level was ("today" or "last time"). 

Also, since the question was this time or last time, is there a need to include 6 levels? If not the left column should be "this time" and Last Time." Or the question on the previous screen should be "do you think you traveled more or less miles today than you did last time on Level 7" Kevin is going to log in to Github and verify

In Bonus Question section - Table says levels, question says levels, dropdown menu has "days" instead of levels

**Comment by TAGraves** [comment]
The third part of this was done in commit e18693904588386e770b4ed528ece8a07edda04f, to be available tomorrow.

Looks like we are still waiting on Kevin for the second part.

The first part is a duplicate of issue #20

**Comment by Jacquelyn-Rudis** [comment]
Spoke with Kevin re: second part:

Remove Levels 4 5 6 7. 

Remove Level 8 and replace with "last time"

Title of left column in table: Day

**Comment by TAGraves** [comment]
This was completed in commit 8f67274759caef4658fd4984ecd748bbc66320e5, available tomorrow.
